### DISCUSSION TITLE
PXE booting from Asus router into netboot.xyz

### DISCUSSION THREAD

**Discussion by Firminator** [antonym/netboot.xyz#1122] [Ideas] [answered]
Lots of people use AsusWRT-Merlin custom firmware for Asus routers. However there is no information out there how to enable PXE booting into netboot.xyz on these devices. Using applicable information from https://programmingflow.com/2015/04/08/boot-any-machine-in-your-home-with-pxe.html and https://netboot.xyz/docs/kb/networking/edgerouter/ I basically frankenstein'd it together and got it working for BIOS and UEFI devices.

@antonym, under the Networking category ( https://netboot.xyz/docs/kb/networking/ ) would you create a second KB article for AsusWRT-Merlin routers? I created a guide with instructions over @ https://github.com/RMerl/asuswrt-merlin.ng/wiki/Enable-PXE-booting-into-netboot.xyz

**Comment by antonym** [comment, marked as answer]
Thanks for the suggestion, added.

https://github.com/netbootxyz/netboot.xyz-docs/commit/b60d2d5b4ea86e4512f82679d994e9782c72d202
